wolfssl/wolfcrypt/error-crypt.h

```diff
@@ -28,9 +28,10 @@
 
 #include <wolfssl/wolfcrypt/types.h>
 
-#ifdef HAVE_FIPS
+#if defined(HAVE_FIPS) && \
+    defined(HAVE_FIPS_VERSION) && (HAVE_FIPS_VERSION < 2)
```

Comment:
HAVE_FIPS_VERSION might not be set in FIPSv1 builds. I think that should be:

    #if defined(HAVE_FIPS) && (!defined(HAVE_FIPS_VERSION) || (HAVE_FIPS_VERSION < 2))

That's what I was doing elsewhere.

Reply:
Thanks, I'll clean that up. I agree.